pkg/networkutils/network.go

```diff
@@ -625,11 +625,11 @@ func getConnmark() uint32 {
 	if connmark := os.Getenv(envConnmark); connmark != "" {
 		mark, err := strconv.ParseInt(connmark, 0, 64)
 		if err != nil {
-			log.Error("Failed to parse "+envConnmark+"; will use ", defaultConnmark, err.Error())
+			log.Errorf("Failed to parse %s; will use %d, error: %v", envConnmark, defaultConnmark, err)
 			return defaultConnmark
 		}
 		if mark > math.MaxUint32 || mark <= 0 {
-			log.Error(""+envConnmark+" out of range; will use ", defaultConnmark)
+			log.Errorf("%s out of range; will use %s", envConnmark, defaultConnmark)
```

Comment:
Use Infof here. The error is being "handled" and therefore you should simply log an informative message that it was handled instead of error.